rundeckapp/grails-app/services/rundeck/services/logging/ThresholdLogWriter.groovy

```diff
@@ -40,13 +47,20 @@ class ThresholdLogWriter extends FilterStreamingLogWriter {
     @Override
     void addEvent(final LogEvent event) {
         def limit = limitReached.get()
+        boolean warning = warningReached.get()
         if (truncate && limit) {
             return
         }
         getWriter().addEvent(event)
 
+        if (!warning && threshold.isWarningSizeReached() && warningReached.compareAndSet(false, true)) {
+            LOG.warn(MSG_WARN_SIZE  , threshold.warningSize, threshold.getValue())
+        }
+
         if (!limit && threshold.isThresholdExceeded() && limitReached.compareAndSet(false, true)) {
-            getWriter().addEvent(LogUtil.logError("Log output limit exceeded: " + threshold.description))
+            String msgError = "Log output limit exceeded: ${threshold.description}"
+            LOG.error(msgError)
```

Comment:
the `LOG.warn` and `LOG.error` do not seem very useful, they will log to the system log, but do not contain any information about which execution has a log log.

Reply:
Done